build-tools/packages/build-cli/src/commands/check/latestVersions.ts

```diff
@@ -0,0 +1,84 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { isInternalVersionScheme } from "@fluid-tools/version-tools";
+import { Args } from "@oclif/core";
+import * as semver from "semver";
+import { findPackageOrReleaseGroup, packageOrReleaseGroupArg } from "../../args.js";
+import { BaseCommand, sortVersions } from "../../library/index.js";
+
+export default class LatestVersionsCommand extends BaseCommand<typeof LatestVersionsCommand> {
+	static readonly summary =
+		"Determines if an input version matches a latest minor release version. Intended to be used in the Fluid Framework CI pipeline only.";
+
+	static readonly description =
+		"This command is used in CI to determine if a pipeline was triggered by a release branch with the latest minor version of a major version.";
+
+	static semverArg = Args.custom({
+		required: true,
+		description: "The version corresponding to the pipeline trigger branch.",
```

Comment:
```suggestion
		description: "The version to check. When running in CI, this value corresponds to the pipeline trigger branch.",
```

I don't think my suggestion is correct - why would a "version" be set to a branch name? So I think you mean something else. But the top-level feedback is not to assume this is ONLY running in CI. Verbose is better in these kinds of descriptions.